main.go

```diff
@@ -5,15 +5,19 @@
 package main
 
 import (
-	"context"
 	"os"
 
 	"github.com/charmbracelet/fang"
 	"github.com/srl-labs/containerlab/cmd"
 )
 
 func main() {
-	if err := fang.Execute(context.TODO(), cmd.RootCmd); err != nil {
+	ctx, _ := cmd.SignalHandledContext()
+
+	cmd.RootCmd.SetContext(ctx)
```

Comment:
actually should check this -- we may not need to do the SetContext here since I would assume (but havent checked yet!) that fang would do that for us. in any case, here is where the context is rooted for all the cmd pkg